src/MCPServer/requirements/base.txt

```diff
@@ -0,0 +1,7 @@
+Django>=1.7,<1.8
+django-mysqlpool==0.1-9
+django-extensions==1.1.1
+mysqlclient==1.3.7
+gearman==2.0.2
+lxml==3.5.0
+pyinotify
```

Comment:
You are right! I made a change in qa/1.x to stop using it. It's funny though that we are declaring a dependency even though that specific code path is never executed. Should we fix that or would that not be a backward-compatible fix?